source/frontend/src/components/common/form/ProjectSelector/ProjectSelector.test.tsx

```diff
@@ -90,13 +90,13 @@ describe('ProjectSelector component', () => {
 
   it('calls onChange callback when a project is selected', async () => {
     const { getInput, findItems } = setup(testForm);
-    await act(() => userEvent.type(getInput()!, 'test'));
+    act(() => userEvent.type(getInput()!, 'test'));
     await waitFor(() => expect(handleTypeaheadSearch).toHaveBeenCalled());
 
     const items = await findItems();
     expect(items).toHaveLength(1);
-    items[0].focus();
-    await act(() => userEvent.keyboard('{Enter}'));
-    expect(onChange).toHaveBeenCalled();
+    act(() => items[0].focus());
+    act(() => userEvent.keyboard('{Enter}'));
+    act(() => expect(onChange).toHaveBeenCalled());
```

Comment:
Usually we don't need to act on "expect" statements - could you try removing the act and see if you get no warnings?

Reply:
When I remove it, it gives the warnings.

Reply:
are you referring to removing the acts on 98? 99? 100? a combination?

Reply:
I only removed the act on line 100 which resulted in warnings.

Reply:
However I did just try replacing the act on line 99 to be an await act(async () => ...) which fixed the warnings so its okay now.